CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 <a name="Pending Release"></a>
 ## [Pending Release](https://github.com/lightstep/otel-launcher-java/compare/0.10.1...master)
+* Add Metrics support
```

Comment:
Also mention we now automatically inject `oshi` instrumentation?

Reply:
added